src/market/directory.rs

```diff
@@ -6,15 +6,15 @@
 
 /// Represents the Tor address and port configuration.
 // It should be set to your specific Tor address and port.
-pub const TOR_ADDR: &str = "127.0.0.1:9050";
+pub const TOR_SOCKS_ADDR: &str = "127.0.0.1:19050";
 
 use bitcoin::Network;
 
 use crate::taker::offers::MakerAddress;
 
 //for now just one of these, but later we'll need multiple for good decentralization
 const DIRECTORY_SERVER_ADDR: &str =
-    "zfwo4t5yfuf6epu7rhjbmkr6kiysi6v7kibta4i55zlp4y6xirpcr7qd.onion:8080";
+    "pl62q4gupqgzkyunif5kudjwyt2oelikpt5pkw5bnvy2wrm6luog2dad.onion:8000";
```

Comment:
This works currently because we are;t really using the directory server right now for fetching maker addresses. Which wasn't an issue for clear net as `localhost` is a constant. But with tor we need a working directory server. For that, the tor address can't be hardcoded.